src/module.ts

```diff
@@ -72,11 +80,24 @@ export default defineNuxtModule<ModuleOptions>({
       },
     })
 
-    // Add plugin to load turnstile script
-    addPlugin({ src: join(runtimeDir, 'plugins/script') })
 
-    // Add <NuxtTurnstile> component
-    addComponentsDir({ path: join(runtimeDir, 'components') })
+    if (shouldUseNuxtScripts) {
```

Comment:
You should register the script with Nuxt Scripts so that it can be optimised at build time, see https://github.com/nuxt/third-party-capital/pull/63/files#diff-030fc083b2cbf5cf008cfc0c49bb4f1b8d97ac07f93a291d068d81b4d1416f70R62

Reply:
Not needed if we don't expose the composable though